src/stcal/jump/jump.py

```diff
@@ -340,50 +380,57 @@ def flag_large_events(gdq, jump_flag, sat_flag, min_sat_area=1,
     :return: none
     """
 
-    log.info('Flagging large events (snowballs, showers).')
+    log.info('TEST Flagging large events (snowballs, showers).')
```

Comment:
Should remove the "TEST" before merging

Reply:
And is this particular function now dedicated to only finding snowballs, with showers in a separate routine? If so, the end of the message should be updated too.